src/main/java/pl/poznan/put/jsontools/logic/JsonTransformFormat.java

```diff
@@ -0,0 +1,37 @@
+package pl.poznan.put.jsontools.logic;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class enables to transform minified JSON file into JSON full form
+ */
+public class JsonTransformFormat implements JsonFinalTransform {
+
+    /**
+     * ObjectMapper to apply formatting to the JSON file
+     */
+    private static final ObjectMapper mapper = new ObjectMapper();
+    /**
+     * Log errors in JsonTransformFormat class
+     */
+    private static final Logger logger = LoggerFactory.getLogger(JsonTransformFormat.class);
+
+    /**
+     * @param json minified JSON file
+     * @return String prettified file (with proper formatting, e.g. spaces, tabs, newlines)
+     */
+    @Override
+    public String execute(JsonNode json) {
+
+        try {
+            return mapper.writerWithDefaultPrettyPrinter().writeValueAsString(json);
+        } catch (JsonProcessingException e) {
+            logger.error("Could not deminify JSON!", e);
+            return null;
+        }
```

Comment:
This can basically be replaced with `json.toPrettyString()`.

Reply:
Upper comment still apply.

`try` block is unnecessary here, as JsonNode is always correct JSON (it's immutable so it can only be modified by casting to subclass and using this subclass's methods).

Reply:
ok, i'll change that